viewer/packages/api/src/public/migration/Cognite3DViewer.ts

```diff
@@ -141,6 +141,7 @@ export class Cognite3DViewer {
   private latestRequestId: number = -1;
   private readonly clock = new THREE.Clock();
   private _clippingNeedsUpdate: boolean = false;
+  private _nearAndFarPlaneNeedsUpdate: boolean = false;
```

Comment:
I think the name for this variable should be different because near and far planes are updated almost every frame because of camera position change. So, I propose something like `_boundingBoxNeedsUpdate`.

Reply:
Roger 👍